src/components/InputTodo.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react"
+
+class InputTodo extends Component {
+  state = {
+    title: "",
+  }
+  onChange = e => {
+    this.setState({
+      [e.target.name]: e.target.value,    
+    })
+  }
+
+  handleSubmit = e => {
+    e.preventDefault();
+    if(this.state.title.trim()) {
+      this.props.addTodoProps(this.state.title);
+      this.setState({    
+        title: ""  
+      });
+    } else {
+      alert("Please write item")
+    } 
+  };
+
+  render() {
+    return (
+      <form onSubmit={this.handleSubmit} className="form-container">
+        <input
+          type="text"
+          className="input-text"
+          placeholder="Add todo..."
+          value={this.state.title}
+          name="title"          
+          onChange={this.onChange}
+        />
+        <button className="input-submit">Submit</button>
+      </form>
+    )
+  }
+}
+export default InputTodo
```

Comment:
Everything looks great and in accordance with the project requirements and would also like to suggest that for every component exported, you should endeavor to close with a semi-colon(;)